flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -1090,9 +1093,9 @@ static void altitude_hold_desired(ManualControlCommandData * cmd, bool flightMod
 		if (cmd->Throttle > DEADBAND_HIGH) {
 			climb_rate = expo3((cmd->Throttle - DEADBAND_HIGH) / (1.0f - DEADBAND_HIGH), altitude_hold_expo) *
 		                         altitude_hold_maxclimbrate;
-		} else if (cmd->Throttle < DEADBAND_LOW && altitude_hold_maxclimbrate > MIN_CLIMB_RATE) {
+		} else if (cmd->Throttle < DEADBAND_LOW && altitude_hold_maxdescentrate > MIN_CLIMB_RATE) {
 			climb_rate = ((cmd->Throttle < 0) ? DEADBAND_LOW : DEADBAND_LOW - cmd->Throttle) / DEADBAND_LOW;
```

Comment:
So say we're at 30% deadband.  DEADBAND_LOW will (now) be .35.   If throttle is less than 0, we will descend at 35% of our maximum descent rate?